cmd/util/tokens.go

```diff
@@ -0,0 +1,70 @@
+package util
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/bacalhau-project/bacalhau/pkg/config"
+	"github.com/bacalhau-project/bacalhau/pkg/config/types"
+	"github.com/bacalhau-project/bacalhau/pkg/storage/util"
+	"github.com/bacalhau-project/bacalhau/pkg/util/closer"
+)
+
+type tokens map[string]string
+
+func readTokens(path string) (tokens, error) {
+	file, err := os.Open(path)
+	if os.IsNotExist(err) {
+		return map[string]string{}, nil
+	} else if err != nil {
+		return nil, err
+	}
+	defer closer.CloseWithLogOnError(path, file)
+
+	var t tokens
+	err = json.NewDecoder(file).Decode(&t)
+	if err != nil {
+		return nil, err
+	}
+
+	return t, nil
+}
+
+func writeTokens(path string, t tokens) error {
```

Comment:
Are we just worried about an ever growing file? With 65535 ports, the max size of the file is only 55Mb! 😉 Yes, who needs to know, and where should I document this? In the docs? (I won't block on getting this answer, but will address in the next commit.)

Reply:
But also, do devstacks on the same port use the same key still? I guess not? So we need some way to handle tokens no longer matching, which I think just results in an authorized state atm instead of a re-authorization.